.github/workflows/test.yml

```diff
@@ -68,6 +68,7 @@ on:
       - 'e2e-tests/**'
       - 'utils/**'
       - 'libs/**'
+      - 'README.md'
```

Comment:
Why has this been added?

Reply:
to trigger tests on README pushes

Reply:
Why do we need tests to run on README pushes?

Reply:
incase we need to artificially trigger tests without a content change so we do it in readme

Reply:
Feels a shame to be running tests and using up time of github runners for changes that only affect the readme though, since there are no changes to any code there can't be failing tests, so we shouldn't need to run the tests should we?

Reply:
ideally for this PR no, but we often have used readme pushes to test the test run on GHA